### DISCUSSION TITLE
ZuluSCSI RP2040 Initiator Mode - LED does not stop blinking

### DISCUSSION THREAD

**Discussion by dgk816** [ZuluSCSI/ZuluSCSI-firmware#129] [Ideas]
Hello,

I just tested out the initiator mode on the ZuluSCSI RP2040.  It worked great.  However, the LED never stops blinking. How are we to know when it's done?  I gave it three hours (to image a 9GB HD) and took a chance that is was done.  According to the log file, it took about 1 hour.

I would suggest making it blink two quick flashes then a short pause and repeat, as a "done" indicator.

Anyway, great product.  I've used several (10+) SCSI2SD and 1 ZuluSCSI (V1.1 Blue) on Sun Sparcstation 20s, and one Sparcstation 2, with great results.

**Comment by PetteriAimonen** [comment]
Yeah, I think it could make sense to change the LED behavior.

Currently after imaging it returns to scan mode, where the LED blinks shortly once a second.
